docs/source/run_model_in_simulation.rst

```diff
@@ -190,15 +230,35 @@ block, results are printed::
           Aladdin Results
     ===============================
 
-Depending on the machine on which the simulation runs, it takes about 10 mins
-on my i7-9850H CPU to finish the simulation.
+How long the simulation will take depends on the machine. On my Intel Core i7
+9850H, it takes about 10 minutes. For larger models, both the trace storage and
+simulation time can become problematic, read on to see the solution.
 
-And congratulations! You just finished running the first SMAUG simulation. The
-:code:`outputs` folder contains simulation stats generated by gem5-Aladdin.
+.. _label_apply_sampling:
+
+Apply sampling to reduce simulation time and trace storage
+----------------------------------------------------------
+
+To save simulation time and trace storage for large models, we can sample the
+loop iterations in the oftentimes repetitive DL kernels. To enable sampling,
+use the :code:`--sample-level` and :code:`--sample-num` parameters in SMAUG.
+:code:`--sample-level` sets the sampling level, which can be one of the five
+levels: :code:`no`, :code:`low`, :code:`medium`, :code:`high` and
+:code:`very_high`. :code:`--sample-num` sets the number of sample iterations
```

Comment:
--sample-num sets the number of iterations a sampled loop will run. A smaller value means more sampling will be applied.

Reply:
Done.